DTS-STN/dts-rmp
# DB Seeder

# Description

A seeder for the Contacts, Engagements and Tags collections, including many-to-many relationships between entities of both. Engagements also have tags attached to them as an array of strings. The collection of tags is for populating lists of recommended tags for autofill.

Use environment variables NUM_CONTACTS, NUM_ENGAGEMENTS to define your own number of Contacts and Engagement documents to be added to the database.

## Test Instructions

1. npm i
2. Add VUE_APP_CONNECTION_STRING=mongodb://localhost:27017/db to your .env
3. npm run mongodb (spin up generic default mongo container)
4. npm run seed (run script for seeding)
5. check data with Robo3T or through mongo shell

## Help Requested

Let me know if there is anything I should add!
